pkg/auth/context.go

```diff
@@ -17,50 +17,52 @@ const (
 	// FilterByOrganisation is used to determine whether resources are filtered by a user's organisation or as an individual owner
 	contextFilterByOrganisation contextKey = "filter-by-organisation"
 	contextIsAdmin              contextKey = "is_admin"
+)
 
+var (
 	// ocm token claim keys
-	ocmUsernameKey string = "username"
-	ocmOrgIdKey    string = "org_id"
-	isOrgAdmin     string = "is_org_admin" // same key used in mas-sso tokens
+	tenantUsernameClaim string = "username"
+	tenantIdClaim       string = "org_id"
+	tenantOrgAdminClaim string = "is_org_admin" // same key used in mas-sso tokens
 
 	// sso.redhat.com token claim keys
-	ssoRHUsernameKey  string = "preferred_username" // same key used in mas-sso tokens
-	ssoRhAccountIdKey string = "account_id"
+	alternateTenantUsernameClaim string = "preferred_username" // same key used in mas-sso tokens
+	tenantUserIdClaim            string = "account_id"
 
 	// mas-sso token claim keys
 	// NOTE: This should be removed once we migrate to sso.redhat.com as it will no longer be needed (TODO: to be removed as part of MGDSTRM-6159)
-	masSsoOrgIdKey = "rh-org-id"
+	alternateTenantIdClaim = "rh-org-id"
 )
 
 func GetUsernameFromClaims(claims jwt.MapClaims) string {
-	if claims[ocmUsernameKey] != nil {
-		return claims[ocmUsernameKey].(string)
+	if claims[tenantUsernameClaim] != nil {
+		return claims[tenantUsernameClaim].(string)
 	}
 
-	if claims[ssoRHUsernameKey] != nil {
-		return claims[ssoRHUsernameKey].(string)
+	if claims[alternateTenantUsernameClaim] != nil {
+		return claims[alternateTenantUsernameClaim].(string)
 	}
-
 	return ""
```

Comment:
I believe this could be rewritten as:
```go
if idx, val := arrays.FindFirst(func(x interface{}) bool { return x != nil }, claims[tenantUsernameClaim], claims[alternateTenantUsernameClaim]); idx != -1 {
	return val.(string)
}
return ""
```